[IMP] employees: larm made targeted edits for 16 and updated pics (#5040)
odoo/documentation: content/applications/hr/employees/new_employee.rst

==================

When a new employee is hired, the first step is to create a new employee form. Starting in the
:menuselection:`Employees` app dashboard, click the :guilabel:`New` button to create a new employee
form. Fill out the required information (underlined in bold) and any additional details.

.. image:: new_employee/new-employee-form.png
   :align: center

>    :alt: Create a new employee card.

Author: I did miss tags! Good catch. As for the rest, I worked in 16.4 for this doc, and I see starting in 16.2 those extra fields are not visible, which is why I did not include them.

  (pencil)` edit icon to select a photo to upload.
- :guilabel:`Job Position`: Enter the employee's job position title, or select the job position from
  the :guilabel:`Job Position` drop-down (see below) to have this field auto-populate. This field
  can be modified and does not need to match the selection made in the :guilabel:`Job Position`
  drop-down.

  .. Example::
     It is recommended to have the job positions match, but if desired, the typed in description in
     this field can contain more information than the selected drop-down job position.

     An example where this may be applicable is if someone is hired for a sales representative
     position configured as :guilabel:`Sales Representative` in the Recruitment app, and that is
     selected for the drop-down :guilabel:`Job Position` field.

     In the typed in job position field beneath the employee's name, the position could be more
     specific, such as `Sales Representative - Subscriptions` if the employee is focused solely on
     subscription sales.

     .. image:: new_employee/job-description-fields.png
        :align: center
        :alt: Both job position fields entered but with different information.

> - Tags: Click on a tag in the drop-down menu to add any tags applicable to the employee. Any tag can

Author: i had this gui-ed originally but it was removed since the word "tag" isn't clickable, it's the tag names. I agreed, so I left it "un-gui-ed"

  be created in this field by typing it in. Once created, the new tag is available for all employee
  records. There is no limit to the amount of tags that can be added.
- Work Contact Information: Enter the employees :guilabel:`Work Mobile`, :guilabel:`Work Phone`,
  :guilabel:`Work Email`, and/or :guilabel:`Company` name.
- :guilabel:`Department`: Select the employee's department from the drop-down menu.
- :guilabel:`Job Position`: Select the employee's job position from the drop-down menu. Once a
  selection is made, the :guilabel:`Job Position` field beneath the employee's name automatically
  updates to reflect the currently selected job position. These positions are from the *Recruitment*
  application, and reflect the currently configured job positions.
- :guilabel:`Manager`: Select the employee's manager from the drop-down menu.
- :guilabel:`Coach`: Select the employee's coach from the drop-down menu.
- :guilabel:`Next Appraisal Date`: This date automatically populates with a date that is computed

>   according to the settings configured in the *Appraisals* application. This date can be modified

Author: Appraisals doc doesn't exist yet, but i linked the recruitment doc to where it says 'recruitment' - great suggestion!

.. tip::
   To make edits to the selected :guilabel:`Department`, :guilabel:`Manager`, :guilabel:`Coach`, or
   :guilabel:`Company`, click the :guilabel:`Internal Link` button next to the respective selection.
   The :guilabel:`Internal Link` button opens the selected form, allowing for modifications. Click

>    :guilabel:`Save` after any edits are made.

Author: In the sentence, it says "to make edits to the selected..." - because the internal link icon only appears after a selection is made. You won't see it when the drop-down is deployed, only after it is selected. Since I give the instruction to make the selection right above- it seems clear.

Resumé
~~~~~~

> Next, enter the employee's work history in the :guilabel:`Resumé` tab. Each previous experience must

Reviewer: ```suggestion
Next, enter the employee's work history in the :guilabel:`Resumé` tab. Each resumé line must
```

     :guilabel:`Levels` field enter `Beginner`, `Intermediate`, and `Expert`, with the
     :guilabel:`Progress` listed as `25`, `50`, and `100`, respectively. Then, either click
     :guilabel:`Save & Close` or :guilabel:`Save & New`.

       .. image:: new_employee/math-skills.png
          :align: center
          :alt: Add new math skills and levels with the skill types form.

Work Information tab
--------------------

The Work Information tab is where the employee's specific job related information is housed. Their
working schedule, various roles, who approves their specific requests (time off, timesheets, and
expenses), and specific work location details are listed here. Enter the following information for
the new employee.

- :guilabel:`Location`: Select the :guilabel:`Work Address` and :guilabel:`Work Location` from the
  corresponding drop-down menus. The work address :guilabel:`Internal Link` button opens up the
  selected company form in a window, and allows for editing. The :guilabel:`Work Location` is the
  specific location details, such as a floor or building. If a new work location is needed, add the
  location by typing it in the field.

> - :guilabel:`Approvers`: Using the drop-down menus, select the employees responsible for approving

Reviewer: ```suggestion
- :guilabel:`Approvers`: Using the drop-down menus, select the users responsible for approving
```
same as with 15, this is a small change, but since you can only choose a user here, not just an employee, I thought it made sense to make that distinction.

- :guilabel:`Planning`: Click on a role from the drop-down menu for both the :guilabel:`Roles` and
  the :guilabel:`Default Role` fields to add a role. There is no limit to the amount of
  :guilabel:`Roles` that can be selected for an employee, but there can only be one
  :guilabel:`Default Role`. The default role is the *typical* role that the employee performs, where
  the :guilabel:`Roles` are *all* the specific roles the employee is able to perform.

  .. image:: new_employee/work-info.png
     :align: center
     :alt: Add the work information to the Work Information tab.

.. tip::
   The employees that appear in the drop-down menu for the :guilabel:`Approvers` section must have
   *Administrator* rights set for the corresponding human resources role. To check who has these
   rights, go to :menuselection:`Settings app --> Users --> → Manage Users`. Click on an employee,
   and check the :guilabel:`Human Resources` section.

   - In order for the employee to appear as an approver for :guilabel:`Expenses`, they must have
     either :guilabel:`Team Approver`, :guilabel:`All Approver`, or :guilabel:`Administrator` set
     for the :guilabel:`Expenses` role.

>    - In order for the employee to appear as an approver for :guilabel:`Time Off`, they must have

Reviewer: ```suggestion
   - In order for the user to appear as an approver for :guilabel:`Time Off`, they must have
```

     either :guilabel:`Officer` or :guilabel:`Administrator` set for the :guilabel:`Time Off` role.

>    - In order for the employee to appear as an approver for :guilabel:`Timesheets`, they must have

Reviewer: ```suggestion
   - In order for the user to appear as an approver for :guilabel:`Timesheets`, they must have
```

Resumé
~~~~~~

Next, enter the employee's work history in the :guilabel:`Resumé` tab. Each resumé line must be
entered individually. Click :guilabel:`Create a new entry`, and the :guilabel:`Create Resumé lines`
form appears. Enter the following information for each entry.

.. image:: new_employee/resume-lines.png
   :align: center
   :alt: Add information for the previous work experience in this form.

- :guilabel:`Title`: Type in the title of the previous work experience.
- :guilabel:`Type`: From the drop-down menu, select either :guilabel:`Experience`,

>   :guilabel:`Education`, :guilabel:`Side Projects`, :guilabel:`Internal Certification`,

Author: I checked this, and it never appears in 16.4,  and I tested about 11 different runbots- I got to see it twice in16.2 and once in a 16.3. I am asking the HR experts since I don't understand why it is visible only so few times. I will update this when I have an naswer.

>   :guilabel:`Education`, :guilabel:`Side Projects`, :guilabel:`Internal Certification`,

Author: Confirmed with the experts: Experience, Education, Side Projects, Internal Certification, an Completed Internal Training are the default options. This has been updated.

Work Information tab
--------------------

The Work Information tab is where the employee's specific job related information is housed. Their
working schedule, various roles, who approves their specific requests (time off, timesheets, and
expenses), and specific work location details are listed here. Enter the following information for
the new employee.

- :guilabel:`Location`: Select the :guilabel:`Work Address` and :guilabel:`Work Location` from the
  corresponding drop-down menus. The work address :guilabel:`Internal Link` button opens up the
  selected company form in a window, and allows for editing. The :guilabel:`Work Location` is the
  specific location details, such as a floor or building. If a new work location is needed, add the
  location by typing it in the field.
- :guilabel:`Approvers`: Using the drop-down menus, select the users responsible for approving
  :guilabel:`Expenses`, :guilabel:`Time Off`, and :guilabel:`Timesheets` for the employee. The
  :guilabel:`Internal Link` button opens a form with the approver's :guilabel:`Name`,
  :guilabel:`Email Address`, :guilabel:`Company`, :guilabel:`Phone`, and :guilabel:`Mobile` fields.

>   These can be modified, if needed.

Author: The employee card is what is created after this whole form is filled in and saved. I don't quite see where this comment applies- I checked the paragraph, a bit above and below, and I think it seems clear- so I am not sure where to add this in this particular section. I will mention this in the beginning/intro so it is understood. 

Also, I am referring to the "card" as the "employee record" since when you click on it it says form - so you fill out the form to make a record. That's the nomenclature I  updated throughout the doc.

- :guilabel:`Schedule`: Select the :guilabel:`Working Hours` and :guilabel:`Timezone` (both
  required) for the employee. The :guilabel:`Internal Link` button opens up a detailed view of the

>   specific daily working hours. Working hours can be modified or deleted here.

Author: Added! It's in Payroll, but is referred to as WORKING SCHEDULES (so confusing!) I added a link to the doc.

- :guilabel:`Planning`: The :guilabel:`Planning` section affects the *Planning* app, and will only
  appear if the *Planning* app is installed. Click on a role from the drop-down menu for both the

>   :guilabel:`Roles` and the :guilabel:`Default Role` fields to add a role. There is no limit to the

Author: In 16.3 they changed it to just default role, and I was using 16.4. I feel this is clear enough for people following along if they are in an earlier version, since only one field says 'default'.

   - In order for the user to appear as an approver for :guilabel:`Expenses`, they must have either
     :guilabel:`Team Approver`, :guilabel:`All Approver`, or :guilabel:`Administrator` set for the
     :guilabel:`Expenses` role.

   - In order for the user to appear as an approver for :guilabel:`Time Off`, they must have either
     :guilabel:`Officer` or :guilabel:`Administrator` set for the :guilabel:`Time Off` role.

   - In order for the user to appear as an approver for :guilabel:`Timesheets`, they must have
     either :guilabel:`Manager`, :guilabel:`Officer`, or :guilabel:`Administrator` set for the
     :guilabel:`Payroll` role.

.. note::
   :guilabel:`Working Hours` are related to a company's working times, and an employee cannot have
   working hours that are outside of a company's working times.

   Each individual working time is company-specific, so for multi-company databases, each company
   needs to have its own working hours set.

   If an employee's working hours are not configured as a working time for the company, new working
   times can be added, or existing working times can be modified. To add or modify a working time,

>    go to the :menuselection:`Payroll app --> Configuration --> Working Times`, and add a new working

Reviewer 2: perfect

   time or edit an existing one.

>    After the new working time is created, set the working hours for the employee.

Reviewer 2: where?

entered. Fields are entered either using a drop-down menu, clicking a check box, or typing in the
information.

- :guilabel:`Private Contact`: Enter the personal :guilabel:`Address` for the employee. The
  selection can be made with the drop-down menu. If the information is not available, type in the
  name for the new address. To edit the new address, click the :guilabel:`Internal Link` button to
  open the address form. On the address form, enter or edit the necessary details.

  Some other information in the :guilabel:`Private Contact` section may auto-populate, if the
  address is already listed in the drop-down menu.

  Enter the employee's :guilabel:`Bank Account Number` using the drop-down menu. If the bank is not
  already configured (the typical situation when creating a new employee) enter the bank account
  number, and click :guilabel:`Create and edit..`. A :guilabel:`Create Bank Account Number` form
  appears. Fill in the information, then click :guilabel:`Save & Close`.

  Then, select the employee's preferred :guilabel:`Language` from the drop-down menu. Next, enter
  the :guilabel:`Home-Work Distance` in the field. This field is only necessary if the employee is

>   receiving any type of commuter benefits. Last, enter the employee's license plate information in

Reviewer 2: ```suggestion
  receiving any type of commuter benefits. Lastly, enter the employee's license plate information in
```

>   the :guilabel:`Private Car Plate` field.

Reviewer 2: I don't see this field present

Author: Private Car Plate first appears in 16.1, so I feel it is universal enough to keep in there.

- :guilabel:`Education`: Select the highest level of education completed by the employee from the
  :guilabel:`Certificate Level` drop-down menu. Options include :guilabel:`Graduate`,
  :guilabel:`Bachelor`, :guilabel:`Master`, :guilabel:`Doctor`, or :guilabel:`Other`. Type in the
  :guilabel:`Field of Study`, and the name of the :guilabel:`School` in the respective fields.
- :guilabel:`Work Permit`: If the employee has a work permit, enter the information in this section.
  Type in the :guilabel:`Visa No` and/or :guilabel:`Work Permit No` in the corresponding fields.

>   Using the calendar module, select the :guilabel:`Visa Expire Date` and/or the :guilabel:`Work

Reviewer 2: Again, this isn't the calendar module!

  Permit Expiration Date` to enter the expiration date(s). If available, upload a digital copy of
  the work permit document. Click :guilabel:`Upload Your File`, navigate to the work permit file in
  the file explorer, and click :guilabel:`Open`.
- :guilabel:`Family Status`: Select either :guilabel:`Single`, :guilabel:`Married`,
  :guilabel:`Legal Cohabitant`, :guilabel:`Widower`, or :guilabel:`Divorced` from the drop-down
  menu. If the employee has any dependent children, enter the :guilabel:`Number of Dependent
  Children` in the field.
- :guilabel:`Emergency`: Type in the name and phone number of the employee's emergency
  contact.

> - :guilabel:`Citizenship`: This section houses all the information relevant to the citizenship of

Reviewer 2: ```suggestion
- :guilabel:`Citizenship`: This section contains all the information relevant to the citizenship of
```

>   the employee. Some selections use a drop-down menu, as does the :guilabel:`Nationality (Country)`,

Reviewer 2: ```suggestion
  the employee. Some selections use a drop-down menu, as does the :guilabel:`Nationality (Country)`,
```

Some selections..... sentence is akward

  :guilabel:`Gender`, and :guilabel:`Country of Birth` sections. The :guilabel:`Date of Birth` uses

>   a calendar module to select the date. First, click on the name of the month, then the year, to

Reviewer 2: not the calendar module

  access the year ranges. Use the :guilabel:`< (left)` and :guilabel:`> (right)` arrow icons,
  navigate to the correct year range, and click on the year. Next, click on the month. Last, click
  on the day to select the date. Type in the information for the :guilabel:`Identification No`,
  :guilabel:`Passport No`, and :guilabel:`Place of Birth` fields.

.. image:: new_employee/private-info.png
   :align: center
   :alt: Add the private information to the Private Information tab.

HR Settings tab
---------------

This tab provides various fields for different information, depending on the country the company is
located. Different fields are configured for different locations, however some sections appear
regardless.

- :guilabel:`Status`: Select an :guilabel:`Employee Type` and, if applicable, a :guilabel:`Related
  User`, with the drop-down menus.

.. important::
   Employees do not also need to be users. Employees do **not** count towards billing, while *Users*
   **do** count towards billing. If the new employee should also be a user, the user must be
   created. After the :guilabel:`User` is created and saved, the new user will appear in the

>    :guilabel:`Related User` field.

Reviewer 2: need to specify that this is their Odoo subscription billing..... can be confusing with invoice app

   After the employee is created, create the user. Click the :guilabel:`⚙️ (gear)` icon, then click
   :guilabel:`Create User`. A :guilabel:`Create User` form appears. Type in the :guilabel:`Name`,

>    :guilabel:`EmaiL Address`, and then select the :guilabel:`Company` from the drop-down menu. Click

Reviewer 2: ```suggestion
   :guilabel:`Email Address`, and then select the :guilabel:`Company` from the drop-down menu. Click
```

>    :guilabel:`Save` after the information is entered.

Reviewer 2: does this automatically link the user and employee? how do I do this manually?

Author: Yes and yes, you can in setting. I referenced the doc to explain how, and added the info that it is automatically linked after a new user card is saved.

- :guilabel:`Payroll`: If applicable, enter the :guilabel:`Registration Number of the Employee` in
  this section. The :guilabel:`Attestation (N-1)` and :guilabel:`Attestation (N)` sections appear
  only for Belgian companies, and will not be visible for other locations. These sections log the
  days that will be paid to the new employee. Enter any :guilabel:`Amount to recover`,
  :guilabel:`Number of days`, and :guilabel:`Recovered Amount` of :guilabel:`Holiday Attest (year) -
  Simple Holiday Pay from previous employer to recover in (year)`, for both N and
  N-1 categories. For the :guilabel:`Holiday Attest (year) - Previous occupation for Double Holiday
  Pay Recovery in (year)` section, click :guilabel:`Add a line`, and enter the number of
  :guilabel:`Months`, the :guilabel:`Amount`, and :guilabel:`Occupation Rate`. Repeat for all

>   entries. Click the :guilabel:`🗑️ (trash can)` icon to delete a line.

Reviewer 2: I don't see payroll in runbot--

Author: It appears to have been added in 16.2.

- :guilabel:`Application Settings`: Enter the employee's :guilabel:`Hourly Cost` in a $XX.XX format.
  This is factored in when the employee is working at a :doc:`work center
  <../../inventory_and_mrp/manufacturing/management/using_work_centers>`. This value affects the

>   manufacturing costs for a product, if the value of the manufactured product is not a fixed amount.

Reviewer 2: you state this in $$ what about European companies for example? 

Also does this impact payroll?

Author: I removed the $  so it is universal. Also, I do not believe it affects payroll at all- I believe this is solely for valuating products. I am checking with the experts on this to be sure

Author: Yup! That's true!

  If applicable, enter the :guilabel:`Fleet Mobility Card` number.
- :guilabel:`SD WORX`: Enter the employee's :guilabel:`SDWorx code` in this field, if applicable.

> - :guilabel:`Attendance/Point of Sale/Manufacturing`: The employee's  :guilabel:`PIN Code` and

Reviewer 2: what is the pin code used for?

Author: Added info- signing in and out of attendance kiosk and POS system.

  :guilabel:`Badge ID` can be entered here, if the employee needs/has one. Click
  :guilabel:`Generate` next to the :guilabel:`Badge ID` to create a badge ID.

> - :guilabel:`Payroll`: Select the employee's :guilabel:`Job Position` from the drop-down menu.

Reviewer 2: again not seeing this at all

Author: Also appeared starting in 16.2

  manufacturing costs for a product, if the value of the manufactured product is not a fixed amount.
  If applicable, enter the :guilabel:`Fleet Mobility Card` number.
- :guilabel:`SD WORX`: Enter the employee's :guilabel:`SDWorx code` in this field, if applicable.
- :guilabel:`Attendance/Point of Sale/Manufacturing`: The employee's  :guilabel:`PIN Code` and
  :guilabel:`Badge ID` can be entered here, if the employee needs/has one. Click
  :guilabel:`Generate` next to the :guilabel:`Badge ID` to create a badge ID.
- :guilabel:`Payroll`: Select the employee's :guilabel:`Job Position` from the drop-down menu.

.. image:: new_employee/hr-settings.png
   :align: center
   :alt: Enter any information prompted in the HR Settings tab for the employee.

Personal documents tab
----------------------

The :guilabel:`Personal Documents` tab is only displayed for certain countries. If this tab is not
visible, it is not applicable to the company's location.
Documents
=========

Add a file for the employee's :guilabel:`ID Card Copy`, :guilabel:`Driving License`,
:guilabel:`Mobile Subscription Invoice`, :guilabel:`SIM Card Copy`, and :guilabel:`Internet

> Subscription Invoice` by clicking the :guilabel:`Upload Your File` button next to the corresponding

Reviewer 2: specify the document name --- let link pull the name.

Author: Done- now it's kind of a mouthful with the repetition of "documents" =D :

For more information on the Documents app, refer to the [Documents] documentation.

~~~~~~

Next, the employee's work history is entered in the :guilabel:`Resumé` tab. Each previous experience

> must be entered individually. Click :guilabel:`Create a new entry`, and the :guilabel:`Create Resumé

Author: This is actually in lower case in the new Milk version. I was also shocked since they are not typically written like this- not sure if this ill be the way they are doing the UI now, but this is correct for the latest V16 version (it could always change back to title case in the next rev, in which case, I'll change it back again =).

   :align: center
   :alt: Add information for the previous work experience in this form.

- :guilabel:`Title`: Type in the title of the previous work experience.
- :guilabel:`Type`: From the drop-down menu, select either :guilabel:`Experience`,
  :guilabel:`Education`, :guilabel:`Side Projects`, :guilabel:`Internal Certification`,
  :guilabel:`Completed Internal Training`, or type in a new entry.
- :guilabel:`Display Type`: Select either :guilabel:`Classic`, :guilabel:`Certification`, or
  :guilabel:`Course` from the drop-down menu.
- :guilabel:`Date Start` and :guilabel:`Date End`: Enter the start and end dates for the work
  experience. To select a date, use the :guilabel:`< (left)` and :guilabel:`> (right)` arrow icons
  to scroll to the desired month, then click on the day to select it.
- :guilabel:`Description`: Enter any relevant details in the field.

.. note::
   After the new employee form is saved, the current position and company is automatically added to
   the :guilabel:`Resumé` tab, with the end date listed as *current*.

Skills
~~~~~~

An employee's skills can be entered in the :guilabel:`Resumé` tab in the same manner a resumé line

> is created. Click the :guilabel:`Create a new entry` button under :guilabel:`Skills` and a

Author: Same here- this is how the button appears, it is written in lower case.

   :align: center
   :alt: Create a new skill for the employee.

- :guilabel:`Skill Type`: Select a :ref:`skill type <employees/skill-types>` by clicking the radio
  button next to the skill type.
- :guilabel:`Skill`: The corresponding skills associated with the selected :guilabel:`Skill Type`
  appear in a drop-down menu. For example, selecting :guilabel:`Language` as the :guilabel:`Skill
  Type` presents a variety of languages to select from under the :guilabel:`Skills` field. Select
  the appropriate pre-configured skill, or type in a new one.
- :guilabel:`Skill Level`: Pre-defined skill levels associated with the selected :guilabel:`Skill
  Type` appear in a drop-down menu. Select a skill level, then the progress bar automatically
  displays the pre-defined progress for that skill level. Skill levels and progress can be modified

>   in the :guilabel:`Skill Level` pop-up form, which is accessed via the :guilabel:`External Link`

Author: Thanks Meng- I could have sworn I saw "External" when it usually say internal in one of the versions , so I don't know why I mixed them up. Maybe it was the 15 version =D Either way, I double checked and yes, these should be internal so I changed them all.

>   in the :guilabel:`Skill Level` pop-up form, which is accessed via the :guilabel:`External Link`

Author: 15 it says External, 16 it says Internal, FYI

  button next to :guilabel:`Skill Level` field.

To delete any line from the :guilabel:`Resumé` tab, click the :guilabel:`🗑️ (trash can)` delete

> icon to delete the entry. Add a new line by clicking the :guilabel:`ADD` button next to the

Author: This is also how the button appears- it looks like they changed the formatting for Milk for these buttons. At least for version 15, they'll be title case/the usual way =)

> icon to delete the entry. Add a new line by clicking the :guilabel:`ADD` button next to the

Reviewer 3: Hi! Just want to reopen this comment to note that whenever buttons are in all caps, we write them in title case for documentation, e.g. "Save & New" :)

> icon to delete the entry. Add a new line by clicking the :guilabel:`ADD` button next to the

Author: Ahh- ok, Fixed!

  Close` to save the level and close the form.

  .. example::
     To add a math skill set, enter `Math` in the :guilabel:`Name` field. Next, in the
     :guilabel:`Skills` field, enter `Algebra`, `Calculus`, and `Trigonometry`. Last, in the
     :guilabel:`Levels` field enter `Beginner`, `Intermediate`, and `Expert`, with the
     :guilabel:`Progress` listed as `25`, `50`, and `100`, respectively. Then, either click
     :guilabel:`Save & Close` or :guilabel:`Save & New`.

       .. image:: new_employee/math-skills.png
          :align: center
          :alt: Add new math skills and levels with the skill types form.

Work information tab
--------------------

- :guilabel:`Location`: Select the :guilabel:`Work Address` and :guilabel:`Work Location` from the
  corresponding drop-down menus. The work address :guilabel:`Internal Link` button opens up the
  selected company form in a window, and allows for editing. The :guilabel:`Work Location` is the
  specific location details, such as a floor, or building. If a new work location is needed, add the
  location by typing it in the field.

> - :guilabel:`Approvers`: Using the drop-down menus, select the employees responsible for approving

Author: I added this info- they need to have Administrator rights set in the HR section of their own user form, in the active users section of the main settings.

payslips and ensure all deductions are accounted for, the employee's personal information should be
entered.

Here, the employee's :guilabel:`Private Contact`, :guilabel:`Education`, :guilabel:`Work Permit`,
:guilabel:`Family Status`, :guilabel:`Emergency Contact`, and :guilabel:`Citizenship` information is
entered. Fields are entered either using a drop-down menu, clicking a check box, or typing in the
information.

> - :guilabel:`Private Contact`: Enter the personal :guilabel:`Address` for the employee. The

Reviewer 3: This is a very long paragraph/bullet point. Consider breaking it up into multiple sub-bullet points beneath the Private Contact bullet point?

  the drop-down menu. If the bank is not already configured (the typical situation when creating a
  new employee) enter the bank account number, and click :guilabel:`Create and edit..`. A
  :guilabel:`Create Bank Account Number` for appears. Fill in the information, then click
  :guilabel:`Save & Close`. Then, select the employee's preferred :guilabel:`Language` from
  the drop-down menu. Next, enter the :guilabel:`Home-Work Distance` in the field. This field is
  only necessary if the employee is receiving any type of commuter benefits. Last, enter the
  employee's license plate information in the :guilabel:`Private Car Plate` field.
- :guilabel:`Education`: Select the highest level of education completed by the employee from the
  :guilabel:`Certificate Level` drop-down menu. Options include :guilabel:`Graduate`,
  :guilabel:`Bachelor`, :guilabel:`Master`, :guilabel:`Doctor`, or :guilabel:`Other`. Type in the
  :guilabel:`Field of Study`, and the name of the :guilabel:`School` in the respective fields.
- :guilabel:`Work Permit`: If the employee has a work permit, enter the information in this section.

>   Type in the :guilabel:`Visa No` and/or :guilabel:`Work Permit No` in the corresponding fields.

Reviewer 3: Does "No" mean "Number" in this case? Would be great to clarify this.

Sidenote, if it does mean "number," these should really have a period after "No." Maybe something you could submit to the feedback team.

>   Using the calendar module, select the :guilabel:`Visa Expire Date` and/or the :guilabel:`Work

Reviewer 3: Also for the feedback team, this should probably read "Visa Expiration Date" in the UI.

Author: Will let them know- thank you!

  the file explorer, and click :guilabel:`Open`.
- :guilabel:`Family Status`: Select either :guilabel:`Single`, :guilabel:`Married`,
  :guilabel:`Legal Cohabitant`, :guilabel:`Widower`, or :guilabel:`Divorced` from the drop-down
  menu. If the employee has any dependent children, enter the :guilabel:`Number of Dependent
  Children` in the field.
- :guilabel:`Emergency`: Type in the name and phone number of the employee's emergency
  contact.
- :guilabel:`Citizenship`: This section houses all the information relevant to the citizenship of
  the employee. Some selections use a drop-down menu, as does the :guilabel:`Nationality (Country)`,
  :guilabel:`Gender`, and :guilabel:`Country of Birth` sections. The :guilabel:`Date of Birth` uses
  a calendar module to select the date. First, click on the name of the month, then the year, to
  access the year ranges. Use the :guilabel:`< (left)` and :guilabel:`> (right)` arrow icons,
  navigate to the correct year range, and click on the year. Next, click on the month. Last, click
  on the day to select the date. Type in the information for the :guilabel:`Identification No`,

>   :guilabel:`Passport No`, and :guilabel:`Place of Birth` fields.

Reviewer 3: The "No"s in this section also need periods at the end, if you're submitting feedback.

Author: I will suggest they add this- I am leaving this as you see it, since this is how it currently appears on the UI.

located. Different fields are configured for different locations, however some sections appear
regardless.

- :guilabel:`Status`: Select an :guilabel:`Employee Type` and, if applicable, a :guilabel:`Related

>   User`, with the drop-down menus.

Reviewer 3: It's not very clear what a "Related User" is.

Author: I added clarification to this since I wasn't entirely sure!

> - :guilabel:`Payroll`: If applicable, enter the :guilabel:`Registration Number of the Employee` in

Reviewer 3: Consider breaking this bullet point up into sub-bullet points, especially the parts that are Belgium-specific.

  this section. The :guilabel:`Attestation (N-1)` and :guilabel:`Attestation (N)` sections appear

>   ony for Belgian companies, and will not be visible for other locations. These sections log the

Reviewer 3: ```suggestion
  only for Belgian companies, and will not be visible for other locations. These sections log the
```

  days that will be paid to the new employee. Enter any :guilabel:`Amount to recover`,
  :guilabel:`Number of days`, and :guilabel:`Recovered Amount` of simple holiday pay, for both N and

>   N-1 categories. For the double holiday pay to recover section, click :guilabel:`Add a line`, and

Reviewer 3: I can't see this in Runbot to verify, but it seems like the "double holiday pay to recover" section should have a guilabel or some other kind of formatting on it?

Author: Yes- I added the full name of the line (it is very long!) for both sections.

  enter the number of :guilabel:`Months`, the :guilabel:`Amount`, and :guilabel:`Occupation Rate`.

>   Repeat for all entries. Click the :guilabel:`🗑️ (trash can)` to delete a line.

Reviewer 3: ```suggestion
  Repeat for all entries. Click the :guilabel:`🗑️ (trash can)` icon to delete a line.
```

- :guilabel:`Application Settings`: Enter the employee's :guilabel:`Hourly Cost` in a $XX.XX format.
  This is factored in when the employee is working at a work center. This value affects the
  manufacturing costs for a product, if the value of the manufactured product is not a fixed amount.
  If applicable, enter the :guilabel:`Fleet Mobility Card` number.
- :guilabel:`SD WORX`: Enter the employee's :guilabel:`SDWorx code` in this field, if applicable.
- :guilabel:`Attendance/Point of Sale/Manufacturing`: The employee's  :guilabel:`PIN Code` and
  :guilabel:`Badge ID` can be entered here, if the employee needs/has one. Click
  :guilabel:`Generate` next to the :guilabel:`Badge ID` to create a badge ID.

> - :guilabel:`Payroll`: Select the employee's :guilabel:`Job Position` from the drop-down menu.

Reviewer 3: Not sure why there's a second Payroll section in your database/image, but in Runbot everything is consolidated into one Payroll section, which makes a lot more sense than having two!

Author: That's odd- both in mt database I made for the tour and app page (started with no data) and the runbot I checked- they both have payroll listed twice (doesn't make sense to me either!) I ept it in the order of the columns as they appear.
![image](https://github.com/odoo/documentation/assets/121518652/7820c805-155c-46e3-9e23-2503f6d407d8)
I will check and see if this is a bug we can have fixed, or if there is a reason this is appearing this way.

Author: OK, ticket is submitted! https://www.odoo.com/web#cids=3&menu_id=4720&action=333&active_id=49&model=project.task&view_type=form&id=3422904

.. image:: new_employee/hr-settings.png
   :align: center
   :alt: Enter any information prompted in the HR Settings tab for the employee.

Documents
=========

All documents associated with an employee are stored in the :guilabel:`Documents` app. The number of

> documents associated with the employee appear in the :guilabel:`Documents` smart button. Click on

Reviewer 3: ```suggestion
documents associated with the employee appear in the :guilabel:`Documents` smart button on the employee form. Click on
```

  the bank is not already configured (the typical situation when creating a new employee) enter the
  bank account number, and click :guilabel:`Create and edit..`. A :guilabel:`Create Bank Account
  Number` form loads. Fill in the information, then click :guilabel:`Save & Close`.

  Then, select the employee's preferred :guilabel:`Language` from the drop-down menu. Next, enter
  the :guilabel:`Home-Work Distance` in the field. This field is only necessary if the employee is
  receiving any type of commuter benefits. Lastly, enter the employee's license plate information in
  the :guilabel:`Private Car Plate` field.
- :guilabel:`Education`: Select the highest level of education completed by the employee from the
  :guilabel:`Certificate Level` drop-down menu. Options include :guilabel:`Graduate`,
  :guilabel:`Bachelor`, :guilabel:`Master`, :guilabel:`Doctor`, or :guilabel:`Other`. Type in the
  :guilabel:`Field of Study`, and the name of the :guilabel:`School` in the respective fields.
- :guilabel:`Work Permit`: If the employee has a work permit, enter the information in this section.
  Type in the :guilabel:`Visa No` and/or :guilabel:`Work Permit No` in the corresponding fields.

>   Using the calendar selector, select the :guilabel:`Visa Expire Date` and/or the :guilabel:`Work

Author: This change is not what is in the UI- they are changing it for V17 as a request on a ticket I made, but I want to keep the text accurate- so that is why I chose to leave it "expire" in this doc.

  resident of the country they are working in, activate the check box next to the
  :guilabel:`Non-resident` field.

HR Settings tab
---------------

This tab provides various fields for different information, depending on the country the company is
located. Different fields are configured for different locations, however some sections appear
regardless.

- :guilabel:`Status`: Select an :guilabel:`Employee Type` and, if applicable, a :guilabel:`Related
  User`, with the drop-down menus. The :guilabel:`Employee Type` options include
  :guilabel:`Employee`, :guilabel:`Student`, :guilabel:`Trainee`, :guilabel:`Contractor`, or
  :guilabel:`Freelancer`.

.. important::
   Employees do not also need to be users. *Employees* do **not** count towards the Odoo
   subscription billing, while *Users* **do** count towards billing. If the new employee should also
   be a user, the user must be created. After the :guilabel:`User` is created and saved, the new
   user will appear in the :guilabel:`Related User` field.

   After the employee is created, create the user. Click the :guilabel:`⚙️ (gear)` icon, then click

>    :guilabel:`Create User`. A :guilabel:`Create User` form appears. Type in the :guilabel:`Name`,

Reviewer 3: ```suggestion
   :guilabel:`Create User`. A :guilabel:`Create User` form appears. Type in the :guilabel:`Name` and
```

   :guilabel:`Email Address`, and then select the :guilabel:`Company` from the drop-down menu. Click
   :guilabel:`Save` after the information is entered, and the employee record is automatically
   updated with the newly created user populating the :guilabel:`Related User` field.

>    User's can also be created manually. For more information on how to manually add a user, refer to

Reviewer 3: ```suggestion
   Users can also be created manually. For more information on how to manually add a user, refer to
```
